### ISSUE TITLE
Update Existing Image parameters null reference exception

### ISSUE THREAD

**Issue by greensweater** [Innofactor/PluginRegistration#72]
When editing pre-image on update step, clicking the ellipsis to edit selected parameters results in exception. CRM 2011 on-premise, UR18. Version 3.2020.4.15

System.NullReferenceException: Object reference not set to an instance of an object.
   at Xrm.Sdk.PluginRegistration.Wrappers.CrmAttribute..ctor(AttributeMetadata md, Boolean isPrimaryId)
   at Xrm.Sdk.PluginRegistration.Helpers.OrganizationHelper.LoadAttributeList(CrmOrganization org, String entityName, ProgressIndicator prog)
   at Xrm.Sdk.PluginRegistration.Controls.CrmAttributeSelectionControl.btnSelect_Click(Object sender, EventArgs e)
   at System.Windows.Forms.Control.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ButtonBase.WndProc(Message& m)
   at System.Windows.Forms.Button.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

**Comment by imranakram** [comment]
I don't have access to any CRM 2011 environment unfortunately. As a workaround you could install an older version and install manually to your XrmToolBox. Here's the URL of a previous nuget https://www.nuget.org/packages/Xrm.Sdk.PluginRegistration/3.2020.1.4

**Comment by greensweater** [comment]
On Wrappers/CrmAttribute.cs recent commit (ba501bc) AttributeTypeName was added. CRM2011 does not include that property in OrgService call.

Null check added to Line 76:
m_attributeTypeName = md.AttributeTypeName != null ? md.AttributeTypeName.Value : null;

Also, line 78 should probably read:
m_validForRead = md.IsValidFor**Read**.Value;

I compiled with this change and plugin is working for me now.
